scripts/lsync.sh

```diff
@@ -43,18 +43,16 @@ fi
 LOCALIZED="$1"
 
 # Try get the English version
-EN_VERSION=`echo $LOCALIZED | sed "s/content\/.\{2,5\}\//content\/en\//g"`
-if [ ! -e $EN_VERSION ]; then
+EN_VERSION=$(echo "$LOCALIZED" | sed "s/content\/.\{2,5\}\//content\/en\//g")
+if [ ! -e "$EN_VERSION" ]; then
   echo "$EN_VERSION has been removed."
   exit 3
 fi
 
 # Last commit for the localized path
-LASTCOMMIT=`git log -n 1 --pretty=format:%h -- $LOCALIZED`
-
-git diff --exit-code $LASTCOMMIT...HEAD $EN_VERSION
+LASTCOMMIT=$(git log -n 1 --pretty=format:%h -- "$LOCALIZED")
 
-if [ "$?" -eq 0 ]; then
+if git diff --exit-code "$LASTCOMMIT...HEAD" -- "$EN_VERSION"; then
```

Comment:
Small comment from a readability standpoint, this line seems densely packed. Would be better to break it down into something like this

```
diff=$(git diff "$LASTCOMMIT" "HEAD" -- "$EN_VERSION")

if [ -z "$diff" ]; then
    echo "$LOCALIZED is still in sync"
    exit 0
fi
```